cooperator_website_recaptcha/controllers/main.py

```diff
@@ -0,0 +1,43 @@
+from odoo.http import request
+from odoo.tools.translate import _
+
+from odoo.addons.cooperator_website.controllers.main import WebsiteSubscription
+
+
+class RecaptchaWebsiteSubscription(WebsiteSubscription):
+    def validation(  # noqa: C901 (method too complex)
+        self, kwargs, logged, values, post_file
+    ):
+        result = super().validation(kwargs, logged, values, post_file)
+        if result is not True:
+            return result
+
+        redirect = "cooperator_website.becomecooperator"
+
+        is_company = kwargs.get("is_company") == "on"
+
```

Comment:
Arf this duplication is painful but I don't see a way around it, the original controller really needs some serious refactoring !